Team Generator Project/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Team Generator</title>
+  </head>
+  <body>
+
+    <header id="header">
+      <h1><a href="">Team Generator</a></h1>      
+      <nav id="nav">
+        <a href="">Reset</a>
+        <a href="">Share this app</a>
+      </nav>
+    </header>
+
+    <main>
+
+      <section>
+
+        <article>
+          <h2 id="aa">Insert the names of the members here</h2>
+          <label>Add a name:</label>
+          <input id="membername" type="text" name="membername"  placeholder="Member Name here" autofocus/>
+          <br/>          
```

Comment:
los </br> son aberraciones de este universo. Ten en cuenta que estás usando HTML para conseguir efectos de CSS

Reply:
Y las id en el caso de los formularios tienen una utilidad añadida que no estás usando. con el atributo "for" del Label si le pones el nombre de la id del input cuando clicken en la Label les pondrá en el input